pro1: Added Polish keyboard layout

Derecho opened the pull request:
As I currently have no way to enter special characters used in the Polish language via longpress, as I'm used to, I leveraged your project and created this Polish keyboard layout based on the existing US International layout with dead Fn keys.

The Polish programmer's keyboard layout is the de-facto standard layout in Poland and is used with regular US (International) QWERTY keyboards. The 6 extra characters in the Polish alphabet are accessible by using AltGr on the respective similarily looking standard characters (except for ź being on x, as ż is already on z). I left the rest of the original US International layout intact as one may want to use this layout for multiple (european) languages.

I'm currently testing this layout on my own device and it appears to work as expected.

anssih commented on finqwerty/src/main/res/values/strings.xml:
```diff
@@ -32,6 +32,7 @@
     <string name="pro1_qwerty_fin_1">FinQwerty F(x)tec Pro1, Finnish for physical US QWERTY</string>
     <string name="pro1_qwerty_hun_1">FinQwerty F(x)tec Pro1, Hungarian for physical US QWERTY</string>
     <string name="pro1_qwerty_nor_1">FinQwerty F(x)tec Pro1, Norwegian for physical US QWERTY</string>
+    <string name="pro1_qwerty_polprog_fndead">FinQwerty F(x)tec Pro1, Polish with Fn dead keys for physical US QWERTY</string>
```
Is there a variant of the Polish programmer's layout that has *direct* (non-Fn/AltGr) dead keys?

If not, I think I'd prefer just dropping the "Fn dead keys", and making the id "pro1_qwerty_pol_prog" or "pro1_qwerty_polprog_1", unless you think the "with Fn dead keys" part is important.

anssih replied:
On another thought, I think I'd like an underscore between pol and prog in any case.

Derecho replied:
As far as I'm aware the Polish Programmer's layout does not have dead keys at all, Fn or not. I based this layout on the usintl_fndead one, as I figured having optional dead keys is just an additional benefit for other european languages, which is why I named it that way. I don't have a preference for the naming, the various options you suggest make sense in their own way. Let me know what you'd like to see and I'll make the change.

anssih replied:
OK, let's make it 'pro1_qwerty_pol_prog' and remove "with Fn dead keys" from the description.

anssih replied:
Also, I think the description should mention programmer... maybe in parantheses like "Polish (programmer)" or in a similar way.

Derecho commented:
I just noticed I forgot one character; the ń on the n key. I shall add this tonight, along with a possible name change of the layout file, and a resolution for the impeding merge conflict.

Derecho commented:
Alright, I've made the following modifications:

- Added the missing ń/Ń character
- Renamed the layout file
- Changed the description of the layout

I'm currently testing a build based on this and it seems to have come out as intended.

anssih commented:
Thanks, looks good now :)
